spring-javaformat/spring-javaformat-checkstyle/src/main/java/io/spring/javaformat/checkstyle/check/SpringJUnit5Check.java

```diff
@@ -38,34 +38,26 @@
  */
 public class SpringJUnit5Check extends AbstractSpringCheck {
 
-	private static final String JUNIT4_TEST_ANNOTATION = "org.junit.Test";
```

Comment:
`org.junit.Test` is used in more than one place so I think this constant should remain.

Reply:
Probably a matter of style, I believe that org.junit.Test is more clear, it represents both where this annotation came from and also give you the exact annotation in this case, also I don't believe it is going to change, but I'll change :)